README.md

```diff
@@ -129,13 +129,13 @@ _Default:_ `null`
 _Default:_ _`''`_ (empty string)
 _Example:_ `'my-project'`
 
-* `navigateFallbackURL`: `string`. The URL being returned from the caches when requested navigation URL isn't available. Similar to `AppCache.FALLBACK` option.  
+* `navigateFallbackURL`: `string`. The URL that should be returned from the cache when a requested navigation URL isn't available within the cache. Similar to the `AppCache.FALLBACK` option.  
```

Comment:
"within the cache or network" since it falls back after trying to load URL from network, i.e.: cache -> network -> fallback